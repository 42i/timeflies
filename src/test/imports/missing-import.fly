# the following line should cause an error:
import this-file-does-not-exist.fly
