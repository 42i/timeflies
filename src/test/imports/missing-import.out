imports/missing-import.fly:2: ERROR : failed to open file; [Errno 2] No such file or directory: 'imports/this-file-does-not-exist.fly'
1 error.
Bill of materials:
imports/missing-import.fly
1 file processed.
